# throwable




# 流程
catch里面是定义异常处理方法的，throw 也是可以用在catch 里面的,也可以用在普通语句抛出异常里面